.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few words)  
```

Comment:
This is obvious from the title no? The idea of the template is to guide the user into creating a better report. Again, the default github template is a lot clearer there:

```suggestion
A clear and concise description of what the bug is.
```